utils/rpms/packaging/get_release_branch

```diff
@@ -2,7 +2,7 @@
 
 # find the base branch of the current branch
 
-set -eux -o pipefail
+set -eu -o pipefail
```

Comment:
@brianjmurrell `-x` is too verbose for the githooks, but FYI in case this extra output was expected somewhere else.